source/puppet/4.1/reference/resources_user_group_windows.markdown

```diff
@@ -18,23 +21,23 @@ These resource types were originally developed for \*nix systems, and have a few
 
 Puppet can use the `user` and `group` resource types to manage **local** accounts.
 
-You can't write a Puppet resource that describes a **domain** user or group. However, a local `group` resource can manage which domain users are members of that local group. See the next section for details.
+You can't write a Puppet resource that describes a **domain** user or group. However, a local `group` resource can manage which domain accounts belong to the local group. See the next section for details.
 
 ### Group Membership
 
-Puppet can use [the `members` attribute][members] of the [`group`][group] type to manage the members of a group.
+Windows can manage group membership by specifying the groups to which a user belongs, or specifying the members of a group. Puppet supports both cases.
 
-A local group can include both local and domain users, so while you can't use Puppet to _manage_ domain users, you can use it to manage their local groups.
+If Puppet is managing a local [`user`][user], you can list the [`groups`][groups] that the user belongs to. Each group can be a local group account, e.g. `Administrators`, or a domain group account.
 
-Groups can also include other groups as members.
+If Puppet is managing a local [`group`][group], you can list the [`members`][members] that belong to the group. Each member can be a local account, e.g. `Administrator`, or a domain account, where each account can be a user or group account.
 
-[members]: /references/4.1.latest/type.html#group-attribute-members
+When managing a `user`, Puppet will make sure the user belongs to all of the `groups` listed in the manifest. If the user belongs to a group not specified in the manifest, Puppet will not remove the user from the group.
 
-You can also use the [auth_membership attribute][auth_membership] to help manage members. Before Puppet 4.1.0, if you had a system with other accounts on it, Puppet would remove any members that weren’t in the manifest. Now with the `auth_membership` attribute set to `false`, the default setting, Puppet will make sure that a group contains as many members as are listed in the manifest. Setting `auth_membership` to `true` makes the list of members in the manifest the authoritative list of members in a group.
+If you want to ensure a `user` belongs to only the `groups` listed in the manifest, and no more, you can specify the [`auth_membership`][auth_membership_user] attribute for the `user`. If set to `inclusive`, Puppet will remove the user from any group not listed in the manifest.
```

Comment:
What @joshcooper mentioned, I had to go look that up - it's all so confusing haha - `membership` is what is used.